Composer/packages/adaptive-form/src/utils/arrayUtils.ts

```diff
@@ -95,7 +98,9 @@ export function useArrayItems<ItemType = unknown>(
   };
 
   const addItem = (newItem: ItemType) => {
-    handleChange(cache.concat(createArrayItem(newItem)));
+    // Allow only the last value in the cache to be empty
+    const newCache = isItemValueEmpty(newItem) ? cache.filter(({ value }) => !isItemValueEmpty(value)) : cache;
+    handleChange(newCache.concat(createArrayItem(newItem)));
```

Comment:
The `AllowedCallers.tsx` component also uses this function. Just checking to make sure that you are aware of this, and that this behavior is ok for all downstream components.

Reply:
Let me double-check and circle back

Reply:
Works for me as before

Reply:
Great, thanks for checking